### ISSUE TITLE
Fix Console Errors

### ISSUE THREAD

**Issue by FinnIckler** [thewca/wca-registration#81]
There are some Errors that appear in the console when you currently use the Frontend. These seem to not affect functionality yet, but we should fix them soon.
Example:
```
Warning: `value` prop on `textarea` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
textarea
RefFindNode2@http://localhost:3002/dist/bundle.js:29403:37
Ref2@http://localhost:3002/dist/bundle.js:29484:20
TextArea2@http://localhost:3002/dist/bundle.js:34375:31
div
RegistrationEditPanel@http://localhost:3002/dist/bundle.js:37436:33
div
Register@http://localhost:3002/dist/bundle.js:37544:52
RenderedRoute@http://localhost:3002/dist/bundle.js:28407:9
Outlet@http://localhost:3002/dist/bundle.js:28649:5
RenderedRoute@http://localhost:3002/dist/bundle.js:28407:9
Outlet@http://localhost:3002/dist/bundle.js:28649:5
main
App@http://localhost:3002/dist/bundle.js:37181:15
RenderedRoute@http://localhost:3002/dist/bundle.js:28407:9
RenderErrorBoundary@http://localhost:3002/dist/bundle.js:28364:7
DataRoutes@http://localhost:3002/dist/bundle.js:28645:9
Router@http://localhost:3002/dist/bundle.js:28659:9
RouterProvider@http://localhost:3002/dist/bundle.js:28598:9
```

**Comment by FinnIckler** [comment]
Might be fixed in #88 and and the latest wca-components release.

**Comment by FinnIckler** [comment]
yeah this got fixed
